test/karma/package.json

```diff
@@ -45,7 +45,7 @@
     "workbox-build": "4.3.1"
   },
   "volta": {
-    "node": "14.21.1",
-    "npm": "6.14.18"
+    "node": "16.20.0",
+    "npm": "8.19.4"
```

Comment:
Can we bump `@types/node` while we're in here? ATM, we're pinned at v14

Reply:
In doing this (bump) I believe that'll update the `package-lock.json` to v2 of the package lock format. If that's the case, can you bump the `package-lock.json` files of under `test/karma` too? I think it's just:
- `test/karma/test-sibling`
- `test/karma/test-invisible-prehydration`